src/librbd/cache/rwl/ImageCacheState.cc

```diff
@@ -83,6 +83,78 @@ void ImageCacheState<I>::dump(ceph::Formatter *f) const {
   ::encode_json("rwl_size", size, f);
 }
 
+static void get_jsonformat(const std::string& s, JSONFormattable *f)
```

Comment:
Nit: wrap in an anonymous namespace instead of using `static`.

Reply:
OK.